editor.planx.uk/src/@planx/components/FileUploadAndLabel/SelectMultiple.tsx

```diff
@@ -281,6 +337,7 @@ export const SelectMultiple = (props: SelectMultipleProps) => {
         aria-live="polite"
         disableClearable
         disableCloseOnSelect
+        disableListWrap
```

Comment:
This was also flagged by @ianjon3s as a usability issue - you can now no longer wrap around from final → first option when navigating by keyboard.